packages/sitecore-jss/src/utils/env.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Parses raw env variable value into a Record
+ * @param {string} mvEnv input
+ * @returns non-empty Record
+ * @throws error when input JSON can't be parsed
+ */
+export const parseMultiValueAsRecord = (mvEnv: string): Record<string, string> => {
+  try {
+    return JSON.parse(mvEnv) as Record<string, string>;
+  } catch (error) {
+    console.log('Parsing of multivalue env variable failed');
+    console.log(`Attempted to parse ${mvEnv}`);
+    throw error;
+  }
+};
+
+/**
+ * Entry method to parse multi or single value env variables
+ * @param {string} envValue - can be undefined when providing values via process.env
+ * @returns Record or string depending on input format
+ */
+export const parseEnvValue = (envValue?: string): Record<string, string> | string => {
```

Comment:
If we're moving this to base package, I think we should aim to make this more generic (not just applicable to POS use-case).

Can we use generics to support a signature something like?:
`tryParseEnvValue<T>(envValue: string | undefined, defaultValue: T): T`

and then use it for POS case like?:
```
const pointOfSale = tryParseEnvValue<Record<string, string>>(process.env.NEXT_PUBLIC_CDP_POINTOFSALE, { [config.defaultLanguage]: process.env.NEXT_PUBLIC_CDP_POINTOFSALE || '' });
```